File: areas.m
%Areas
inicio=0;
while inicio==0;
disp('Calculo de áreas');
disp('1. Cuadrado');
disp('2. Círculo');
disp('3. Triángulo');
disp('4. Rectángulo');
opcion = input('Seleccione el area: ');

archivo=fopen('Areas.txt','a');
switch opcion
    case 1, disp('Cuadrado--------------------------');
        x = input('ingrese el lado: ');
        resul = x*x;
        fprintf('resultado=: %d\n',resul);
        fprintf(archivo,'El área es: %d\n',resul);
        back = input('Desea Continuar s/n: ','s');
            if back=='s'
                inicio=0;
                %clc
            elseif back=='n'
                inicio=1;
            end
    case 2, disp('Circulo------------------------------');
        x = input('ingrese el radio del circulo: ');
        resul = pi*x*x;
        %disp('resultado: %d\n',resul);
        fprintf('resultado=: %d\n',resul);
        fprintf(archivo,'El área es: %d\n',resul);
            back = input('Desea Continuar s/n: ','s');
            if back=='s'
                inicio=0;
                %clc
            elseif back=='n'
                inicio=1;
            end
    case 3, disp('Triángulo---------------------');
        x = input('ingrese la base: ');
        y = input('ingrese la altura: ');
        resul = x*y/2;
        %disp('resultado: %d\n',resul);
        fprintf('resultado=: %d\n',resul);
        fprintf(archivo,'El área es: %d\n',resul);
            back = input('Desea Continuar s/n: ','s');
            if back=='s'
                inicio=0;
                %clc
            elseif back=='n'
                inicio=1;
            end
    case 4, disp('Rectángulo------------------------');
        x = input('ingrese la base: ');
        y = input('ingrese el ancho: ');
              
        resul = x*y;
        %disp('resultado: %d\n',resul);
        fprintf('resultado=: %d\n',resul);
        fprintf(archivo,'El área es: %d\n',resul);
            back = input('Desea Continuar s/n: ','s');
            if back=='s'
              inicio=0;
              %clc
            elseif back=='n'
              inicio=1;
            end
           
     otherwise, disp('Operación No Valida');
end
end
fclose(archivo);

File: areas.py
archivo=open("Areas.txt", "a")
import math

print("Calcular Áreas de figuras Geométricas.\n")
print("1.Cuadrado.\n2.Circulo.\n3.Triángulo.\n4.Rectángulo.\n")
x=int(input("Escoja la figura: "))
    
if x==1:
    L=int(input('Ingrese el lado: '))
    area=L*L
    print("El área del cuadrado es: ", area)
    archivo.write("Área del cuadrado es: " + 'area=% s' %area + '\n')
    
if x==2:

    R=int(input('Ingrese el radio del circulo: '))
    area=math.pi*R**2
    print("El área del círculo es: ", area)
    archivo.write("Área del circulo es: " + 'area=% s' %area +'\n')
    
if x==3:
    
    b=int(input('Ingrese la base: '))
    h=int(input('Ingrese la altura: '))
    area=b*h/2
    print("El área del triángulo es: ",area)
    archivo.write("Área del triángulo es: " + 'area=% s' %area + '\n')
  
   
if x==4:
    
    b=int(input('Ingrese la base: '))
    h=int(input('Ingrese la altura: '))
    area=b*h
    print("El área del rectángulo es: ",area)
    archivo.write("Área del rectángulo es: " + 'area=% s' %area + '\n')

archivo.close ()
    
